src/content/docs/blog/2025/07/11/Development update: os-tools/index.md

```diff
@@ -0,0 +1,184 @@
+
+In our testing, the code showed significant speedups across all three of our supported file systems (xfs, ext4 and f2fs). The previous single-threaded blitting made using ext4 and f2fs particularly slow, to the point that we did not recommend users use either filesystem as the basis of an AerynOS install.
+
+However, blitting speeds with the new parallel approach -- particularly with a "cold" kernel vfs cache -- have significantly improved. Whilst ext4 and f2fs are still not as performant as xfs for our use case, they are at least more serviceable as the basis of an AerynOS install than they used to be. By way of an example, I saw a ~2x blitting speed improvement on my Gen4 NVME SSD using xfs with the new parallel blitting code.
+
+It's worth restating that, to our knowledge, the moss approach to [atomic updates](https://aerynos.com/blog/2025/03/29/aerynos-the-os-as-infrastructure/#%EF%B8%8F-atomic-updates), is the only one of its kind (at least in the Linux space) where users do not have to rely on containerization or A/B system swaps to deliver package updates. Eliminating download speeds as a variable, Moss is capable of atomically installing/updating hundreds of packages on your system in a matter of seconds to tens of seconds on SSD drives, and the installed/upgraded applications are ready to use next time the application is opened. No reboots and no messing with container permissions necessary.
+
+Given that boulder also needs to blit files when it creates buildroots, the code has also had a positive impact on reducing package build times. This will be more evident on larger package builds and will have a cumulative impact, the more package work you end up doing.
+
+### Moss: Sync available before installed packages
+
+As we were testing the upgrade path from our old `packages.aerynos.com/volatile` repository to our new, CDN-backed `cdn.aerynos.dev/unstable` repository, we ran into some unexpected small niggles related to how packages are resolved.
+
+While tarkah's [fix](https://github.com/AerynOS/os-tools/pull/514) to Moss was relatively small in terms of code, it served to ensure that updates would install properly on the first go when syncing to the new repo.
+
+Consequently, we synced this bug-fix to the Moss version in the old repository to ensure that users will be able to seamlessly upgrade to the new rolling `unstable` repository.
+
+
+### Moss: Add index output directory option
+
+As we were preparing the process for sync'ing the packages in our `volatile` build-server repository to our new downstream rolling `unstable` repository on our public-facing server, we ran into an issue with the existing `moss index` code path.
+
+Before this issue was fixed, the `stone.index` file would be unconditionally written next to the actual package .stone files. This was useful when indexing local repos, but not as useful when indexing actual stone [pool/](https://cdn.aerynos.dev/unstable/pool/) directories and sub-directories.
```

Comment:
```suggestion
Before this issue was fixed, the `stone.index` file would be unconditionally written next to the actual package `.stone` files. This was useful when indexing local repos, but not as useful when indexing actual stone [pool/](https://cdn.aerynos.dev/unstable/pool/) directories and sub-directories.
```

Not sure about the link there. Seems like something is wrong. Was it meant to be [stone pool](https://cdn.aerynos.dev/unstable/)?